src/parser/warrior/arms/modules/core/Execute/ExecuteRange.js

```diff
@@ -15,14 +17,16 @@ const EXECUTE_RANGE_MASSACRE = 0.35;
  */
 class ExecuteRangeTracker extends Analyzer {
```

Comment:
tbh should probably just swap this to use the ExecuteHelper module

Reply:
I figured, I saw Kill Shot for hunters was using the helper.

There was a lot I wanted to clean up in this class but I figured I would get this small change in first and then submit a larger PR that would update all of that. Also, since this class technically works I didn't know if we wanted to change everything to use the helper.